miqa/core/tasks.py

```diff
@@ -204,15 +240,28 @@ def perform_export(project_id: Optional[str]):
 
     for project_object in projects:
         for frame_object in Frame.objects.filter(scan__experiment__project=project_object):
-            data.append(
-                [
-                    project_object.name,
-                    frame_object.scan.experiment.name,
-                    frame_object.scan.name,
-                    frame_object.scan.scan_type,
-                    frame_object.frame_number,
-                    frame_object.raw_path,
+            row_data = [
+                project_object.name,
+                frame_object.scan.experiment.name,
+                frame_object.scan.name,
+                frame_object.scan.scan_type,
+                frame_object.frame_number,
+                frame_object.raw_path,
+            ]
+            last_decision = frame_object.scan.decisions.last()
+            if last_decision:
```

Comment:
A code comment explaining this encoding scheme would also be a good idea.

Reply:
Added in [0ed474f](https://github.com/OpenImaging/miqa/pull/392/commits/0ed474f3cad06e516fd43dc99574db1cc8a55d7f)